playground/backend/cmd/server/controller.go

```diff
@@ -15,6 +15,7 @@
 package main
 
 import (
+	"beam.apache.org/playground/backend/internal/emulators"
```

Comment:
minor: unnecessary move of import package. Wondering that doesn't playground code have go fmt check?